src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
Following a brief but instructive email exchange with @edani, I have come up with this prototype to illustrate what I think is possible... https://github.com/lemire/dictionary (all the mistakes are my own)